src/senselab/audio/tasks/speaker_diarization/api.py

```diff
@@ -29,6 +29,9 @@ def diarize_audios(
     Returns:
         List[List[ScriptLine]]: The list of script lines with speaker labels.
     """
+    if model is None:
+        model = PyannoteAudioModel(path_or_uri="pyannote/speaker-diarization-3.1", revision="main")
```

Comment:
Can you please specify more clearly in the doc string what's the behavior with None? This comment applies for all other cases

Reply:
Good point. I'll address that